### ISSUE TITLE
Get RunDirectory from topic, proposal and run number

### ISSUE THREAD

**Issue by zhujun98** [European-XFEL/karabo_data#94]
Martin from SCS proposed to be able to get a RunDirectory instance by given topic, proposal number and run number, which simplifies the input and can also hide the implementation of the '/gpfs...' folder hierachy. For example

```
run = RunDirectory('SCS', '900048', run='3')
``` 
In this case, I think we may also need some convenient functions like

```
get_proposals('SCS')

get_runs('SCS', '900048')
```
which is something like a database query.

**Comment by takluyver** [comment]
I think it makes sense to have a way to get the run without the full path, e.g. to make it easy to iterate through runs. I'd create a separate function for it, rather than overloading `RunDirectory`. Maybe `load_run('SCS', '900048', 3)`? 

I'm not so convinced about `get_proposals` and `get_runs`. The filesystem is easy to understand, and people either already know `ls` or will benefit more from learning it than they will from learning special-purpose functions for this. I don't think hiding the filesystem is a useful goal.

**Comment by zhujun98** [comment]
`load_run(topic, proposal, run_number)` is a good convenient function signature. Since there are another layer of directory, i.e. `201831`, `201801`, some small extra work will need to find the location of the proposal folder.

I agree that `get_proposals` and `get_runs` maybe redundant. We should have another database API to do this.

**Comment by CFGrote** [comment]
just fyi, i prototyped sth along these lines in https://git.xfel.eu/gitlab/xtooldev/XTR/blob/master/spb-sfx/RunAnalysis.ipynb

**Comment by tmichela** [comment]
closed with #147
